tests/json/schema-1-gradient/input.py

```diff
@@ -57,7 +57,7 @@
   "return_energy": -76.02139738600329
 }
 
-psi4.json_wrapper.run_json(json_data)
+json_data = json_data = psi4.json_wrapper.run_json(json_data)
```

Comment:
two `=`, huh?

Reply:
Something is going wonky somewhere. I dont see this on my laptop. Lets pull it in and check it out. It doesn't actually hurt anything, but we should patch it if we see it post-merge.

Reply:
I see `json_data = json_data = psi4.json-wrapper....`